packages/osd-interpreter/package.json

```diff
@@ -27,7 +27,7 @@
     "del": "^6.1.1",
     "getopts": "^2.2.5",
     "pegjs": "0.10.0",
-    "sass-loader": "npm:@amoo-miki/sass-loader@10.4.1-node-sass-9.0.0-libsass-3.6.5",
+    "sass-loader": "npm:@amoo-miki/sass-loader@10.4.1-node-sass-9.0.0-libsass-3.6.5-with-sass-embedded.rc1",
```

Comment:
This patch is getting increasingly gross 🤢

Reply:
since it's an npm for, can we give it a more friendly name, also are we able to keep `sass-loader` up to date since the official one is already at `13.3.2` https://github.com/webpack-contrib/sass-loader/tree/v13.3.2, it might cause future troubles in case we need to bump `sass-loader` version down the road

Reply:
@ZilongX we can only use sass-loader@9 with our ancient webpack@4.